src/__test__/unit/encoders.test.ts

```diff
@@ -102,23 +97,25 @@ describe('encoders', () => {
     });
 
     test('addKvRecords', () => {
+      const fwConstants = buildFirmwareConstants();
       const mockObject = {
-        client: buildMockConnectedClient(),
         type: 0,
         records: { key: 'value' },
         caseSensitive: false,
+        fwConstants,
       };
       const payload = encodeAddKvRecordsRequest(mockObject);
       const payloadAsString = payload.toString('hex');
       expect(payloadAsString).toMatchSnapshot();
     });
 
     test('removeKvRecords', () => {
+      const fwConstants = buildFirmwareConstants();
       const mockObject = {
-        client: buildMockConnectedClient(),
         type: 0,
-        ids: [0],
+        ids: ['0'],
```

Comment:
This is a `uint32_t[]` type in firmware. Not sure why you're making it into a string

Reply:
If we want to follow firmware definition, that should be changed to `ids: number[]`

Reply:
Made an issue to make that breaking change at some point in the future: https://github.com/GridPlus/gridplus-sdk/issues/522